draft-store.yml

```diff
@@ -0,0 +1,12 @@
+version: '2'
```

Comment:
Please exclude this file change.

Reply:
This change can stay however it is not recommended to be committed.